indexer/services/comlink/public/api-documentation.md

```
@@ -4136,6 +4136,99 @@ fetch(`${baseURL}/turnkey/signin`,
+  "code": "string"
+}
+```
+
+### Parameters
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|body|body|[AppleLoginRedirectRequest](#schemaappleloginredirectrequest)|true|none|
+
+> Example responses
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)